examples/convert_mri_and_trans.py

```diff
@@ -0,0 +1,168 @@
+t1_mgh = nib.load(t1_mgh_fname)
+t1_nii_fname = op.join(data_path, 'subjects', 'sample', 'mri', 'T1.nii.gz')
+nib.save(t1_mgh, t1_nii_fname)
+
+# Load the transformation matrix and show what it looks like
+trans_fname = op.join(data_path, 'MEG', 'sample',
+                      'sample_audvis_raw-trans.fif')
+trans = mne.read_trans(trans_fname)
+print(trans)
+
+###############################################################################
+# We can save the MRI to our existing BIDS directory and at the same time
+# create a JSON sidecar file that contains metadata, we will later use to
+# retrieve our transformation matrix :code:`trans`.
+
+# We use the write_anat function
+write_anat(bids_root=output_path,  # point to the BIDS dir we wrote earlier
+           subject=sub,
+           t1w=t1_nii_fname,  # path to the MRI scan in NIfTI format
+           session=ses,
+           raw=raw,  # the raw MEG data file connected to the MRI
+           trans=trans,  # our transformation matrix
+           verbose=True  # this will print out the sidecar file
+           )
+
+# Clean up the NIfTI file we wrote earlier to convert from mgh to NIfTI
+os.remove(t1_nii_fname)
+
+# Let's have another look at our BIDS directory
+print_dir_tree(output_path)
+
+###############################################################################
+# Our BIDS dataset is now ready to be shared. We can easily reproduce our
+# transformation matrix using ``MNE-BIDS`` and the BIDS dataset.
+
+bids_fname = bids_basename + '_meg.fif'
+
+# reproduce our trans
+_trans = get_head_mri_trans(bids_fname=bids_fname,  # name of the MEG file
+                            bids_root=output_path  # root of our BIDS directory
+                            )
+
+# Show that they are the same
+np.testing.assert_almost_equal(trans['trans'],
+                               _trans['trans'])
+
+###############################################################################
+# Finally, let's use the T1 weighted MRI image and plot the anatomical
+# landmarks Nasion, LPA, and RPA (=left and right preauricular points) onto
+# the brain image. For that, we can extract the location of Nasion, LPA, and
+# RPA from the MEG file, apply our transformation matrix :code:`trans`, and
+# plot the results.
+
+# Get Landmarks from MEG file
+coords_dict = _extract_landmarks(raw.info['dig'])
+pos = np.asarray((coords_dict['LPA'],
+                  coords_dict['NAS'],
+                  coords_dict['RPA']))
+
+# Apply transform
+mri_pos = apply_trans(trans, pos) * 1e3
+
+# Get fiducials in voxel space
+vox2ras_tkr = t1_mgh.header.get_vox2ras_tkr()
+ras2vox_tkr = np.linalg.inv(vox2ras_tkr)
+vox2ras = t1_mgh.header.get_vox2ras()
+mri_pos = apply_trans(ras2vox_tkr, mri_pos)  # in vox
+mri_pos = apply_trans(vox2ras, mri_pos)  # in RAS
+
+# Plot it
+fig, axs = plt.subplots(3, 1)
+for i in range(3):
+    plot_anat(t1_mgh_fname, axes=axs[i], cut_coords=mri_pos[i, :])
```

Comment:
nitpick but we normally discourage the use of single letter variable name. The reason is pragmatic -- you can't Ctrl + F them easily.